src/frontend/src/pages/CreateProjectPage/CreateProjectFormView.tsx

```diff
@@ -16,7 +16,7 @@ import { yupResolver } from '@hookform/resolvers/yup';
 import { CreateProjectFormInputs } from './CreateProjectForm';
```

Comment:
On Line 7 you imported 'Button' but it was never used, so you might want to delete this line. Any time you import something but never use it, you should delete it.